testproject/users/migrations/0004_user_insert_document_id_user_delete_document_id.py

```
@@ -0,0 +1,43 @@
+# Generated by Django 5.1.15 on 2025-12-24 14:29
+
+import pgtrigger.compiler
+import pgtrigger.migrations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0003_alter_user_profile"),
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)